### PR TITLE
JS Core WG Roadmap

### PR DESCRIPTION
**Pull request by daviddias**
Preview: https://github.com/ipfs/roadmap/blob/roadmap-wg-integration-web-browsers/WG_JS_CORE.md

### CODE REVIEW

**Comment by daviddias** [comment]
Hi JS Core Team!

Now that the Roadmap is a live PR on Github, it is a good time to check in on our Roadmap Instruction Manuals and check in to see if the Roadmap contemplates everything we could have wished for and everything we believe it is valuable for the community to have access. Think things like: Is the Vision described? Are the Requirements well understood? We have a good sense of priority and dependencies? 

You can find the Roadmap Instruction Manuals at:
- https://github.com/ipfs/team-mgmt/blob/master/ROADMAP_HOW_TO.md
- https://github.com/ipfs/team-mgmt/blob/master/TEAM_COORDINATION_TOOLKIT.md#types-of-roadmaps

Check in with your WG and have fun giving this review! :)

File: WG_JS_CORE.md [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+📦 = Package Managers  🗂 = Large Files  🔄 = Decentralized Web  🤝 = Partner Use Cases  🧠 = Strategic Projects
+ 
+- JS IPFS is ready for use in production. It is stable, performant, and interoperable with other IPFS implementations 📦 🗂 🔄 🤝 🧠
+- JS IPFS can ingest and move data with similar speeds to rsync or wget 📦 🗂 🔄 🤝
+- Data on JS IPFS is searchable both online and offline 📦 🔄 🤝
+- Developer experience is a joy  🔄 🤝 🧠
+
+## 💎 Milestones
+
+- `M (P0)`: Production ready 📦 🗂 🔄 🤝 🧠. JS IPFS has for a long time now been under heavy development with a focus on implementing features. We’re now at a stage where it has a good level of feature parity to Go IPFS. Implementing the last key missing features and focused attention on performance, stability and security issues will allow JS IPFS to be a viable alternative to Go IPFS.
```
Comment (alanshaw) [inline WG_JS_CORE.md]: Can these please be sub-headings so that we can link to them?

Comment (daviddias) [reply WG_JS_CORE.md]: Absolutely. Take full ownership of adjusting the doc as you see fit :)

**Comment by momack2** [comment]
Merging this PR to document where we are on our roadmaps - future iterations to narrow down on a more focused and achievable set of roadmap initiatives (to match project-level priority update) will follow in next few weeks.

**Pull request merged**